doc/api/tty.md

```diff
@@ -289,6 +289,10 @@ The `tty.isatty()` method returns `true` if the given `fd` is associated with
 a TTY and `false` if it is not, including whenever `fd` is not a non-negative
 integer.
 
+Within `worker_threads`, this API will return `false` for process.stdout.
+This is because the `process.stdout` of the worker is a pipe into the parent,
+not actually a TTY.
```

Comment:
This is not accurate:

```js
> void new Worker(`console.log('isatty(1)', require('tty').isatty(1))`, {eval:true})
undefined
> isatty(1) true
```

`tty.isatty()` is generally not concerned with what thread it’s being called on, as `fd`s are a process-wide resource.

Reply:
@addaleax , I am describing  `fd 1`, instead process.stdout.fd
```js
void new Worker(`console.log('isatty(process.stdout.fd)', require('tty').isatty(process.stdout.fd))`, {eval:true})
undefined
satty(process.stdout.fd) false
```

Reply:
Edit: ~~I am  describing fd 1~~ ->  I am NOT describing fd 1

Reply:
`process.stdout.fd` is `undefined` in a worker thread. (Should it be?)

```console
> void new Worker('console.log("fd:", process.stdout.fd)', {eval: true})
undefined
> fd: undefined
```